tests/e2e/test_driver.py

```diff
@@ -52,6 +53,7 @@ def __init__(self, method_name):
         # If running in local mode, just use environment variables for params.
         self.arguments = os.environ if get_args_from_env else {}
         self.arraysize = 1000
+        self.buffer_size_bytes = 10485760
```

Comment:
Does this need to be updated per this comment on your other open pull request?

https://github.com/databricks/databricks-sql-python/pull/151#discussion_r1251176373

Reply:
I actually modified it in L114 in this file for `test_cloud_fetch` and was keeping `self.buffer_size_bytes = 10485760` to keep it as before for the existing e2e tests. I'll update this value to the new default